### DISCUSSION TITLE
not able to digitalize object

### DISCUSSION THREAD

**Discussion by SimbiosiGT** [alicevision/AliceVision#1025] [Q&A]
I start to play with Meshroom 2021.1.0 the piece was black and I didn't manage to capture the object so I painted white but obtain no result.
I can digitalize the entourage but not the object intended with 18 shots with DSLR.
Could you advise what I doing wrong?
![image](https://user-images.githubusercontent.com/56190579/114692384-dc8df980-9d18-11eb-86ce-afdf5b75c652.png)

**Comment by natowi** [comment]
Try splash paint it with water colour for example or draw on it with markers. Pure white is as bad as pure black :) Also get far closer to your object.

**Comment by SimbiosiGT** [comment]
Thank you will test more
